mono/metadata/marshal.c

```diff
@@ -4257,11 +4256,17 @@ mono_marshal_get_struct_to_ptr (MonoClass *klass)
 	if (marshal_info->str_to_ptr)
 		return marshal_info->str_to_ptr;
 
-	if (!stoptr) {
+	if ((res = marshal_info->str_to_ptr))
+		return res;
```

Comment:
Duplicating logic from the preceeding `if`. Guessing you meant to delete one of them.

Reply:
Yeah it was a "bad" merge/rebase, thank you.

Reply:
(or checkout -p of old revision, similar)